src/components/Chat/ChatMessage.tsx

```diff
@@ -2,8 +2,21 @@ import * as React from 'react'
 import * as PropTypes from 'prop-types'
 
 import { childrenExist, createShorthandFactory, customPropTypes, UIComponent } from '../../lib'
+import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
+import { Extendable, ReactChildren } from '../../../types/utils'
+import Avatar, { IAvatarProps } from '../Avatar/Avatar'
+export interface IChatMessageProps {
+  as?: any
+  avatar?: IAvatarProps
```

Comment:
If you agree to go with shorthand factory avatar type will change as well.

Reply:
done